views/pages/StudiesPage/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+
+import { Box } from '@mui/material'
+
+import { SORT_DIRECTION } from '../../../constants'
+import api from '../../api'
+import PageHeader from '../../components/PageHeader'
+import useTableSort from '../../hooks/useTableSort'
+import StudiesTable from '../../tables/StudiesTable'
+
+const siteName = 'siteName'
+const study = 'study'
+
+const StudiesPage = () => {
+  const [studies, setStudies] = useState([])
+  const { onSort, sortDirection, sortBy } = useTableSort('study')
```

Comment:
You can use the `study` var here right?

Reply:
Oof good point. 
[2e93642](https://github.com/AMP-SCZ/dpdash/pull/723/commits/2e9364246f77a5ff0a140abfa602e9fd409157ae)